pkg/prm/exec.go

```diff
@@ -10,7 +12,23 @@ const (
 )
 
 // Executes a tool with the given arguments, against the codeDir.
-func (*Prm) Exec(toolName string, args []string) (ExecExitCode, error) {
-	// TODO
-	return EXEC_ERROR, nil
+func (*Prm) Exec(tool *Tool, args []string) error {
+
+	if tool.Cfg.Gem != nil {
+		fmt.Printf("GEM")
+	}
+
+	if tool.Cfg.Puppet != nil {
+		fmt.Printf("PUPPET")
+	}
+
+	if tool.Cfg.Binary != nil {
+		fmt.Printf("PUPPET")
```

Comment:
Should this be `PUPPET`?

Reply:
yep

Reply:
I think the print statement should be `BINARY` in this case to match the others.